<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShortcutEdit</class>
 <widget class="QDialog" name="ShortcutEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add Shortcut</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>19</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leName">
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbType">
         <item>
          <property name="text">
           <string>exec</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>focus</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>move</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>workspace</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>splith</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>splitv</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>layout</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>fullscreen</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>scratchpad</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>mode</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>shrink</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>grow</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>reload</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>kill</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Keybinding</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leBinding"/>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Clause</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leClause"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="cbLocked">
     <property name="text">
      <string>Allow when locked</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>19</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbSave">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>ShortcutEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>744</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>795</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
